Documentation/install/guides/kops.rst

```diff
@@ -0,0 +1,31 @@
+**********************************
+Kubernetes Kops Installation Guide
+**********************************
+
+As of kops 1.9 release, Cilium is supported as CNI plugin for kops-deployed clusters.
+
+Cilium needs newer kernel version than default kops images, so you need to supply an ami which will be new enough for Cilium to run on it.
+
+CoreOS images have new enough kernels for Cilium to run on them and are tested by kops developers, which makes them a perfect candidate.
+
+The latest stable CoreOS AMI can be found using aws cli:
+
+.. code:: bash
+
+    $ aws ec2 describe-images --region=us-east-1 --owner=595879546273 \
+        --filters "Name=virtualization-type,Values=hvm" "Name=name,Values=CoreOS-stable*" \
+        --query 'sort_by(Images,&CreationDate)[-1].{id:ImageLocation}'
+
+You also need to change default etcd version run in kops, as Cilium needs at least version 3.1.0 (kops default is still on 2.0 branch).
```

Comment:
... change the default etcd version used in kops...

Reply:
Done